examples/digital_fingerprinting/production/morpheus/dfp/utils/derive_args.py

```diff
@@ -161,14 +176,31 @@ def _set_mlflow_tracking_uri(self):
         mlflow.set_tracking_uri(self._tracking_uri)
         logger.info("Tracking URI: %s", mlflow.get_tracking_uri())
 
+    def _set_time_fields(self):
+        if self._is_train_and_infer or self._is_training or self._is_inference:
+            self._time_fields = self._create_time_fields(self._duration)
+        else:
+            raise Exception(
```

Comment:
In general I think raising the base `Exception` class should be avoided.

```suggestion
            raise ValueError(
```